### PR TITLE
Refactor map

### PR DESCRIPTION
**Pull request by nilsjung**
* change map rendering
* change signal strenght display

### CODE REVIEW

File: frontend/components/Map/Map.js [hunk 1]
```diff
@@ -1,59 +1,62 @@
...
+        // add users position to list
+        if (this.currentPositionHasChanged()) {
+            const position = (
+                <Marker
+                    title="your position"
+                    key="own-data-marker"
+                    coordinate={this.props.location}
+                />
+            );
+            this.setState({ currentPositionMarker: position });
```
Comment (karsil) [inline frontend/components/Map/Map.js:40]: Should this be `this.setState({...this.state, currentPositionMarker: position });` ?

File: frontend/components/Map/Map.js [hunk 2]
```diff
@@ -1,59 +1,62 @@
...
+        }
         this.setState({ marker: dataMarker });
```
Comment (karsil) [inline frontend/components/Map/Map.js:42]: S.o.

**Review by karsil — COMMENTED** [review]
Looks fine, only a question about the use of `setState`

File: frontend/components/Map/Map.js (hunk 1 above)
Comment (nilsjung) [reply frontend/components/Map/Map.js:40]: Should be done by the ‘setState‘ method internally

**Review by karsil — APPROVED** [review]

**Pull request merged**